setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = dv-utils
-version = 1.0.0
```

Comment:
1.0.3 is the latest version, not sure how it got out-of-sync (bad rebasing probably). Will have to bump this anyway when releasing, so you can ignore this